numba/tests/test_unicode.py

```diff
@@ -1277,6 +1277,24 @@ def pyfunc(x):
             self.assertEqual(pyfunc(*args), cfunc(*args),
                              msg='failed on {}'.format(args))
 
+    def test_isalpha(self):
+        def pyfunc(x):
+            return x.isalpha()
+
+        cfunc = njit(pyfunc)
+        # Samples taken from CPython testing:
+        # https://github.com/python/cpython/blob/201c8f79450628241574fba940e08107178dc3a5/Lib/test/test_unicode.py#L738-745     # noqa: E501
+        cpython = ['\uD800', '\uDFFF', '\uD800\uD800', '\uDFFF\uDFFF',
+                   'a\uD800b\uDFFF', 'a\uDFFFb\uD800',
+                   'a\uD800b\uDFFFa', 'a\uDFFFb\uD800a']
+        # https://github.com/python/cpython/blob/1d4b6ba19466aba0eb91c4ba01ba509acf18c723/Lib/test/test_unicode.py#L613-L621    # noqa: E501
```

Comment:
This URL refers to `test_isspace`, this code is checking `isalpha`

Reply:
Fixed